.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: '[BUG]'
+labels: 'bug'
+assignees: ''
+
+---
+
+Please make sure to follow & check below prerequisites.
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this bug has not already been reported
+
+## Describe the Bug
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Steps to reproduce the behavior:
+1. ...
+2. ...
+
+## Expected Behavior
+A clear and concise description of what you expected to happen.
+
+## Additional Context
+Add any other context about the problem here.
+
+- [ ] I will abide by the [code of conduct](https://github.com/bhavik2936/dot-files/blob/main/.github/CODE_OF_CONDUCT.md).
```

Comment:
Let's correct this link to current repo instead of `dot-files`.

Reply:
Please check the updated changes.